### DISCUSSION TITLE
Does 'Insert Vector Count Rate' in Milvus monitoring record deletion operations of data?

### DISCUSSION THREAD

**Discussion by Saxon0918** [milvus-io/milvus#26388] [Q&A and General discussion]
I would like to inquire the developers of Milvus whether 'Insert Vector Count Rate' in the monitoring of Milvus database records the deletion operations of the data?
![image](https://github.com/milvus-io/milvus/assets/26076141/9c99e88a-2708-41d2-a167-73b5ad741ea2)

**Comment by yhmo** [comment]
“Insert Vector Count Rate” is only for insertion.

There is another metric "Success Request Rate", you will see delete requests rate if you have deletions.
![Screenshot from 2023-08-16 16-39-53](https://github.com/milvus-io/milvus/assets/2282099/76b9a99d-0ef7-4eaf-841c-f261201653ec)

**Reply by Saxon0918** [reply]
Thanks for your answer.
